web-nfc/index.js

```diff
@@ -1,4 +1,4 @@
-scanButton.addEventListener("click", async () => {
+  scanButton.addEventListener("click", async () => {
```

Comment:
I don't think this is needed right?

Reply:
I did nto modify this line..

Reply:
I meant the 2 spaces added. Sorry for the confusion.